content/release-notes/1.22.0.md

```diff
@@ -0,0 +1,17 @@
+---
+date: 2020-10-30
+linktitle: "1.22.0"
+title: "1.22.0"
+kubernetes: "1.17, 1.18, and 1.19"
+---
+
+{{<features>}}
+  * Added support for disaster recovery backups and restores. Restores can be created both on the same instance or on a different instance. Embedded clusters do not support this feature yet.
+{{</features>}}
+
+{{<fixes>}}
+* Fixed a bug where base64 encoded application icons do not render in airgapped installations.
+* Fixed a total snapshot capture time.
+* Fixed remaining seconds when restoring volumes from application backups.
+* Fixed a bug in the KOTS Admin Console UI where a failure from "Check for updates" was not properly indicated.
```

Comment:
Hmm... this doesn't make much sense. What's the failure? Is this about better error reporting?

Reply:
i believe so yeah